.github/CONTRIBUTING.md

```diff
@@ -290,3 +305,14 @@ Git can append the `Signed-off-by` line to your commit messages. Simply
 provide the `-s` or `--signoff` option when performing a `git commit`.
 For more information about writing commit messages, visit [How to Write
 a Git Commit Message](https://chris.beams.io/posts/git-commit/).
+
+#### Co-authored By
+If someone else had part in your pull request, please add the following to the commit:
+`Co-authored-by: Name <gitregistered@email.adress>`
+This is useful if their authorship was lost during squashing, rebasing, etc.,
+but may be used in any situation where there are co-authors.
+
+The email adress used here should be the same as on the GitHub profile of said user.
```

Comment:
adress -> address

Reply:
GitHub vs Github is something thats debatable :P
Don't personally mind either 👍